scripts/scil_bundle_fixel_analysis.py

```diff
@@ -0,0 +1,291 @@
+                   help='List of the names of the bundles, in the same order '
+                        'as they were given. \nIf this argument is not used, '
+                        'the script assumes that the name of the bundle \nis '
+                        'its filename without extensions.')
+
+    p.add_argument('--dps_key', default=None, type=str,
+                   help='Key to access the data per streamline to use as '
+                        'weight when computing the maps, \ninstead of the '
+                        'number of streamlines. [%(default)s].')
+
+    g = p.add_argument_group(title='Mask parameters')
+
+    g.add_argument('--abs_thr', default=0, type=float,
+                   help='Value of density maps threshold to obtain density '
+                        'masks, in number of streamlines \nor streamline '
+                        'weighting if --dps_key is given. Any number of '
+                        'streamlines \nor weight above this value will '
+                        'pass the absolute threshold test [%(default)s].')
+
+    g.add_argument('--rel_thr', default=0.01, type=float,
+                   help='Value of density maps threshold to obtain density '
+                        'masks, as a ratio of the normalized density '
+                        '\nAny normalized density above '
+                        'this value will pass the relative threshold test. '
+                        '\nMust be between 0 and 1 [%(default)s].')
+
+    g.add_argument('--norm', default="voxel", choices=["fixel", "voxel"],
+                   help='Way of normalizing the density maps. If fixel, '
+                        'will normalize the maps per fixel, \nin each voxel. '
+                        'If voxel, will normalize the maps per voxel. '
+                        '[%(default)s]')
+
+    p.add_argument('--max_theta', default=45,
+                   help='Maximum angle between streamline and peak to be '
+                        'associated [%(default)s].')
+
+    p.add_argument('--split_bundles', action='store_true',
+                   help='If set, save the density maps for each bundle '
+                        'separately \nin addition to the all in one version.')
+
+    p.add_argument('--split_fixels', action='store_true',
+                   help='If set, save the density maps for each fixel '
+                        'separately \nin addition to the all in one version.')
+
+    p.add_argument('--single_bundle', action='store_true',
+                   help='If set, will save the single-fiber single-bundle '
+                        'masks as well. \nThese are obtained by '
+                        'selecting the voxels where only one bundle is '
+                        'present \n(and one fiber/fixel).')
+
+    p.add_argument('--bundles_mask', action='store_true',
+                   help='If set, save the bundle mask for each bundle.')
+
+    add_overwrite_arg(p)
+    add_processes_arg(p)
```

Comment:
The -v option does not seem to do much... Could be cool to add a bit of verbosing

Reply:
It's better. Can you not write something in the help along the lines of ... 
''In terms of running time, this script will run in approximately X minutes with a standard output of tractoflow/rbx_flow bundles" 

In my case, X seems to be around 15-30min.

Reply:
Je vois que c'est l'étape de 
Computing fixel density for all bundles...
la plus longue...

Reply:
On my setup, it takes about 2 minutes for 33 bundles (with 8 threads). It it super dependent on the number of streamlines I think... I can add an upper bound time.